src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -786,8 +786,9 @@ private void OnClusterTransactionCompletion(ClusterTransactionCommand.SingleClus
                     {
                         if(t.IsFaulted)
                         {
-                            RachisLogIndexNotifications.NotifyListenersAbout(index, t.Exception);
-                            ServerStore.Cluster.ClusterTransactionWaiter.SetException(options.TaskId, t.Exception);
+                            Exception e = t.Exception.InnerException;
```

Comment:
Why are you assuming that there is an inner exception here? What if there isn't?

Reply:
Done